index.html

```diff
@@ -0,0 +1,113 @@
+                        <li class="Judge row col-12 col-sm-12"><h2 class="col-4 col-sm-4">Arbitrations</h2><p class="col-8 col-sm-8">Ordis thinks that the operator has...wasted time...INVESTED a lot in farming vitus essence.</p></li>
+                        <li class="Judge row col-12 col-sm-12"><h2 class="col-4 col-sm-4">Normal Star Chart</h2><p class="col-8 col-sm-8">The Lotus must be so proud of the operator, maybe Ordis will meet a similar fate someday.</p></li>
+                        <li class="Judge row col-12 col-sm-12"><h2 class="col-4 col-sm-4">Steelpath Star Chart</h2><p class="col-8 col-sm-8">Operator we can not let Teshin be forgotten. QUITE NOW...Carry out his dream.</p></li>
+                        <li class="Judge row col-12 col-sm-12"><h2 class="col-4 col-sm-4">Archon Hunts</h2><p class="col-8 col-sm-8">Ordis thinks the operator looks...like a porcupine...SPLENDID with the shards installed .</p></li>
+                    </ul></li>
+                <li class="Planned col-lg-4 offset-lg-4 col-8 offset-2 col-sm-8 offset-sm-2">SEE WHAT ORDIS HAS PLANNED FOR THE OPERATOR</li>
+                </ul>
+          </section>
+          <h2 class="More col-10 col-sm-10 offset-1 offset-sm-1">View More</h2>
+          <section id="FooterTop">
+            <div class="nav_top col-lg-12 col-12 col-sm-12">
+                <div class="Platform col-lg-2 col-2 col-sm-2 offset-5">Platforms</div>
+                <div class="col-lg-2 offset-lg-5 col-2 col-sm-2 offset-sm-5 offset-5"><hr class="Line col-lg-2 offset-lg-5 col-2 col-sm-2 offset-sm-5 offset-5"></div>
+                <ul class="nav_top row col-lg-12 col-12 col-sm-12">
+                    <li class="col-lg-2 offset-1"><img class="Console col-lg-9" src="images/PC.jpg" alt="PC"></li>
+                    <li class="col-lg-2"><img class="Console col-lg-9" src="images/PS4.jpg" alt="PS4"></li>
+                    <li class="col-lg-2"><img class="Console col-lg-9" src="images/PS5.jpg" alt="PS5"></li>
+                    <li class="col-lg-2"><img class="Console col-lg-9" src="images/xboxone.jpg" alt="xboxone"></li>
+                    <li class="col-lg-2"><img class="Console col-lg-9" src="images/seriesx.jpg" alt="seriesx"></li>
+                </ul>
+            </div>
+          </section>
+          <section id="FooterBottom">
+            <div class="nav_top row col-lg-12 col-12 col-sm-12">
+                <div class="col-lg-2 col-2 col-sm-2 offset-1"><img class="col-lg-5 col-5 col-sm-5" src="images/Warframelogo.jpg" alt="Warframelogo"></div>
+                <ul class="nav_top col-lg-6 col-6 col-sm-6 offset-1">
+                    <li class="col-lg-12 col-12 col-sm-12">NINJAS PLAY FREE</li>
+                    <li class="col-lg-12 col-12 col-sm-12">Calculating intelligence profile of the grineer...ERROR: not a number...did the operator enjoy this witticism.</li>
+                </ul>
+            </div>
+          </section>
+      <script src="script.js"></script>
```

Comment:
- [ ] These two sections are supposed to be in the footer tag because it's a footer, please kindly fix that 🙃